go/src/koding/klientctl/machine.go

```diff
@@ -33,33 +41,28 @@ func tabFormatter(w io.Writer, infos []*machine.Info) {
 	now := time.Now()
 	tw := tabwriter.NewWriter(w, 2, 0, 2, ' ', 0)
 
-	fmt.Fprintf(tw, "TEAM\tSTACK\tPROVIDER\tLABEL\tOWNER\tAGE\tIP\tSTATUS\n")
+	fmt.Fprintf(tw, "ID\tALIAS\tTEAM\tSTACK\tPROVIDER\tLABEL\tOWNER\tAGE\tIP\tSTATUS\n")
 	for _, info := range infos {
-		fmt.Fprintf(tw, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
-			info.Team,
-			info.Stack,
-			info.Provider,
+		fmt.Fprintf(tw, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
+			info.ID,
+			info.Alias,
+			unknownIfEmpty(info.Team),
+			unknownIfEmpty(info.Stack),
+			unknownIfEmpty(info.Provider),
 			info.Label,
 			info.Owner,
-			timeToAgo(info.CreatedAt, now),
+			machine.ShortDuration(info.CreatedAt, now),
 			info.IP,
-			prettyStatus(info.Status),
+			machine.PrettyStatus(info.Status, now),
 		)
 	}
 	tw.Flush()
 }
 
-func prettyStatus(status machine.Status) string {
-	now := time.Now()
-
-	if status.State == machine.StateOnline {
-		return fmt.Sprintf("%-9s (%s)", status.State, timeToAgo(status.ModifiedAt, now))
-	}
-
-	timeReasonFmt := timeToAgo(status.ModifiedAt, now)
-	if status.Reason != "" {
-		timeReasonFmt += ": " + status.Reason
+func unknownIfEmpty(val string) string {
+	if val == "" {
+		return "<unknown>"
```

Comment:
I will change this to `-`. However, a bunch of `-` characters can also look strange when it's many of them. <unknown> is also consistent with Unknown state string.

I added this because of your post on `kd channel`.
```
$ kd machine ls
TEAM           STACK                PROVIDER  LABEL        OWNER  AGE       IP             STATUS
team           koding.com           aws       development         26d 20h   54.82.165.242  online    (26d 11h)
banana-test01  IFTTT demo           aws       ifttt_2             196d 2h                  offline   (196d 2h)
mashape-kong   Mashapekong's Stack  aws       kong-1              124d 13h                 offline   (124d 10h: Machine is stopped)
test23473      Rjeczalik's Stack    aws       example             160d 6h                  offline   (160d 5h: Machine is stopped)
rrccgg         Gokmen's Stack       aws       testvm              190d 3h                  offline   (190d 3h)
barbar555555   Rjeczalik's Stack    aws       example_1           143d 3h                  offline   (143d 2h: Machine is stopped)
-              -                    aws       example_1           142d 23h                 offline   (142d 22h: Machine is stopped)
banana-test01  IFTTT demo           aws       ifttt_1             196d 2h                  offline   (196d 2h)
```

vs

```
$ kd machine ls
TEAM           STACK                PROVIDER  LABEL        OWNER  AGE       IP             STATUS
team           koding.com           aws       development         26d 20h   54.82.165.242  online    (26d 11h)
banana-test01  IFTTT demo           aws       ifttt_2             196d 2h                  offline   (196d 2h)
mashape-kong   Mashapekong's Stack  aws       kong-1              124d 13h                 offline   (124d 10h: Machine is stopped)
test23473      Rjeczalik's Stack    aws       example             160d 6h                  offline   (160d 5h: Machine is stopped)
rrccgg         Gokmen's Stack       aws       testvm              190d 3h                  offline   (190d 3h)
barbar555555   Rjeczalik's Stack    aws       example_1           143d 3h                  offline   (143d 2h: Machine is stopped)
<unknown>      <unknown>            aws       example_1           142d 23h                 offline   (142d 22h: Machine is stopped)
banana-test01  IFTTT demo           aws       ifttt_1             196d 2h                  offline   (196d 2h)
```